tools/mo/openvino/tools/mo/main.py

```diff
@@ -95,6 +95,15 @@ def print_argv(argv: argparse.Namespace, is_caffe: bool, is_tf: bool, is_mxnet:
     print('\n'.join(lines), flush=True)
 
 
+def get_default_frontends():
+    # Set which frontend to use by default, values should be 'new' or 'legacy'
+    default_frontends = {
+        'onnx': 'legacy',
```

Comment:
are we going to switch to new ONNX frontend in a separate PR?

Reply:
yes, it is connected with some more activities, more details in 75535